lang/cs/Org.Apache.REEF.Wake/Remote/Proto/WakeRemoteProtosGen.cs

```diff
@@ -29,27 +29,27 @@
 // Generated from: src/main/proto/RemoteProtocol.proto
 namespace Org.Apache.REEF.Wake.Remote.Proto
 {
-  [global::System.Serializable, global::ProtoBuf.ProtoContract(Name=@"WakeMessagePBuf")]
+  [global::System.Serializable, global::ProtoBuf.ProtoContract(Name = @"WakeMessagePBuf")]
```

Comment:
We probably want to exclude files generated from .proto as well?

Also, we have 4 files generated from .avro under Org.Apache.REEF.Common.Avro, I'd exclude them now as well. It's strange that StyleCop doesn't complain about them already, they have the same line "//<auto-generated />" as the other ones...